### DISCUSSION TITLE
problems with depth, but can't reproduce

### DISCUSSION THREAD

**Discussion by MrDiablon** [solidjs/solid-router#383] [General]
Hello, I was updating my modules, and to validate everything is ok i launch a test with vitest, but I got an error.

![image](https://github.com/solidjs/solid-router/assets/15982079/c0db2e3e-29df-4e69-88c9-e3ad8714829b)

I have this error with version 11 and further .
The environnement for my test is happy-dom.

I have try to launch a simple test  like `2+2=4` but got the error before going throw it so it's not my test.

I have try to create a reproduction but not get it.

It's look like that the code to replace the `state` of `window.history` is not working. 

I don't no if it's only me or if someone else got this error, so I post here before creating an issue maybe I have miss something, so if you can help me, I thank's you

**Comment by sophie-richardson25** [comment]
I am seeing something similar after using the SolidJS <A> tag in one of my components, testing it with Vitest leads to the same error. Are there any updates on this?

**Reply by MrDiablon** [reply]
Hi, no update here, because I haven't got responce I haven't create an issue, I moke the window object to put a value int the "_depth" variable

**Comment by MrDiablon** [comment]
the problems is no longer present
